### PR TITLE
Improve MSRV docs.

### PR DESCRIPTION
**Pull request by xStrom**
This PR does minor wording adjustments to the MSRV notice and also moves the Cargo command to its own line.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -231,14 +231,19 @@ VELLO_STATIC_LOG="vello=trace" VELLO_STATIC_ARGS="--test-scenes" cargo apk run -
...
-This version of Vello has been verified to compile with Rust 1.75 and later.
-
-Note that since then, one of Vello's dependencies might have released a new version with a higher requirement.
-Thus if you encounter a compilation issue due to a dependency, you should downgrade it via `cargo update -p package_name --precise 0.1.1` (with a real name and version).
+This version of Vello has been verified to compile with **Rust 1.75** and later.
 
 Future versions of Vello might increase the Rust version requirement.
 It will not be treated as a breaking change and as such can even happen with small patch releases.
 
+> [!TIP]
```
Comment (DJMcNab) [inline README.md:239]: See [#linebender > GitHub (markdown) alerts](https://xi.zulipchat.com/#narrow/stream/419691-linebender/topic/GitHub.20.28markdown.29.20alerts) on using these alerts. It has some unfortunate consequences

File: README.md [hunk 2]
```diff
@@ -231,14 +231,19 @@ VELLO_STATIC_LOG="vello=trace" VELLO_STATIC_ARGS="--test-scenes" cargo apk run -
...
+> As time has passed, some of Vello's dependencies could have released versions with a higher Rust requirement.
+> If you encounter a compilation issue due to a dependency and don't want to upgrade your Rust toolchain, then you could downgrade the dependency.
+> ```sh
+> # Use the problematic dependency's name and version
+> cargo update -p package_name --precise 0.1.1
+> ```
```
Comment (DJMcNab) [inline README.md:245]: I'm not completely convinced that we actually need this whole section to give this much detail. It seems like too much detail for those who don't need this, but too little detail for anyone who actually does (e.g. why do we have dependencies which can do these kinds of publish?)

Can we make "downgrade the dependency" a link to cargo update's doc?

**Review by DJMcNab — APPROVED** [review]
This is an improvement in the formatting, so approving. But I'm not convinced the content is actually that useful to have front-and-center

File: README.md (hunk 1 above)
Comment (xStrom) [reply README.md]: I don't have strong feelings about this syntax, so I'll remove it from this PR so that discussion on the syntax can remain separated.

**Review by DJMcNab — APPROVED** [review]

**Pull request merged**